src/Notepads/Services/FileExtensionProvider.cs

```diff
@@ -197,6 +198,11 @@ public static class FileExtensionProvider
 
         public static bool IsFileExtensionSupported(string fileExtension)
         {
+            if ((int)SystemInformation.OperatingSystemVersion.Build >= 19041)
```

Comment:
Add comments and link to your question under reunion repo as well for quick reference.